CHANGELOG_PENDING.md

```diff
@@ -14,10 +14,12 @@ program](https://hackerone.com/tendermint).
 - Apps
 
 - Go API
+- [libs] \#3811 Remove `db` from libs in favor of `https://github.com/tendermint/tm-cmn`
```

Comment:
I don't think we should make a new `tm-cmn` repo. It should just be `db` or `tm-db` and only have the db code.

It seems the only code removed in the PR was the `db` package, which is great, but putting it as a pkg in a new  `tm-cmn` seems misleading ...